### PR TITLE
Fixes for uninstall

### PR DESCRIPTION
**Pull request by jsuchome**

### CODE REVIEW

File: installer/kfserving/description.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: kfserving
 description: |
   Serverless Inferencing on Kubernetes
-namespace: kfserving-system
+namespace:
```
Comment (stefannica) [inline installer/kfserving/description.yaml:4]: Does the field have to be present, even if it doesn't contain a value ?

Comment (jsuchome) [reply installer/kfserving/description.yaml:4]: It does not

**Comment by flaviodsr** [comment]
I can confirm that it fixes the skipping `mlflow` part, note that it is also removing the `fuseml-workloads` namespace which deletes everything inside it such as tekton tasks, inference services, etc... But I guess that is ok since we will be moving mlflow to its own namespace.

Regarding `kantive` the installer fails with the following message:
```Failed to uninstall extension kfserving: failed to uninstall using kustomize directory https://github.com/fuseml/extensions/installer/kfserving/kustomize: kubectl apply failed:
namespace "kfserving-system" deleted
Warning: apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
customresourcedefinition.apiextensions.k8s.io "inferenceservices.serving.kubeflow.org" deleted
customresourcedefinition.apiextensions.k8s.io "trainedmodels.serving.kubeflow.org" deleted
Warning: admissionregistration.k8s.io/v1beta1 MutatingWebhookConfiguration is deprecated in v1.16+, unavailable in v1.22+; use admissionregistration.k8s.io/v1 MutatingWebhookConfiguration
mutatingwebhookconfiguration.admissionregistration.k8s.io "inferenceservice.serving.kubeflow.org" deleted
Warning: admissionregistration.k8s.io/v1beta1 ValidatingWebhookConfiguration is deprecated in v1.16+, unavailable in v1.22+; use admissionregistration.k8s.io/v1 ValidatingWebhookConfiguration
validatingwebhookconfiguration.admissionregistration.k8s.io "inferenceservice.serving.kubeflow.org" deleted
validatingwebhookconfiguration.admissionregistration.k8s.io "trainedmodel.serving.kubeflow.org" deleted
serviceaccount "kfserving-models-web-app" deleted
role.rbac.authorization.k8s.io "leader-election-role" deleted
clusterrole.rbac.authorization.k8s.io "kfserving-manager-role" deleted
clusterrole.rbac.authorization.k8s.io "kfserving-models-web-app-cluster-role" deleted
clusterrole.rbac.authorization.k8s.io "kfserving-proxy-role" deleted
rolebinding.rbac.authorization.k8s.io "leader-election-rolebinding" deleted
clusterrolebinding.rbac.authorization.k8s.io "kfserving-manager-rolebinding" deleted
clusterrolebinding.rbac.authorization.k8s.io "kfserving-models-web-app-binding" deleted
clusterrolebinding.rbac.authorization.k8s.io "kfserving-proxy-rolebinding" deleted
configmap "inferenceservice-config" deleted
configmap "kfserving-config" deleted
configmap "kfserving-models-web-app-config-9kkt28dhgb" deleted
secret "kfserving-webhook-server-secret" deleted
service "kfserving-controller-manager-metrics-service" deleted
service "kfserving-controller-manager-service" deleted
service "kfserving-models-web-app" deleted
service "kfserving-webhook-server-service" deleted
deployment.apps "kfserving-models-web-app" deleted
statefulset.apps "kfserving-controller-manager" deleted
virtualservice.networking.istio.io "kfserving-models-web-app" deleted
Error from server (NotFound): error when deleting "https://github.com/fuseml/extensions/installer/kfserving/kustomize": the server could not find the requested resource (delete certificates.cert-manager.io serving-cert)
Error from server (NotFound): error when deleting "https://github.com/fuseml/extensions/installer/kfserving/kustomize": the server could not find the requested resource (delete issuers.cert-manager.io selfsigned-issuer)
: exit status
```

**Comment by jsuchome** [comment]
Ah, removing namespace by extension should not be done when it is using the default namespace.

The second problem is weird, I think this should be fixed by https://github.com/fuseml/fuseml/pull/180/commits/24ef63042d7b70f8a47a6619a3077d6e5d330b75 ... I have to investigate again

**Comment by jsuchome** [comment]
@flaviodsr however, both items you mention cannot be solved in this repo, so I think this PR actually does not need any changes

**Review by flaviodsr — APPROVED** [review]

**Pull request merged**